lib/storage/src/content_manager/toc.rs

```diff
@@ -621,12 +628,22 @@ impl TableOfContent {
         };
         if let Some(on_apply) = on_apply {
             if on_apply.send(result.clone()).is_err() {
-                log::warn!("Failed to notify on collection meta operation completion.")
+                log::warn!("Failed to notify on consensus operation completion.")
```

Comment:
is the content of the error never interesting to log?

Reply:
Generally the send error is always that a receiver is dropped. But yes, let's add it for clarity.

Reply:
Yeah, in `Err(_)` is just returns the value that it failed to send, so it doesn't make sense to print it. But I added some clarification.